projects/go-lib/src/lib/components/go-icon-button/go-icon-button.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'go-icon-button',
+  templateUrl: './go-icon-button.component.html'
+})
+export class GoIconButtonComponent {
+  @Input() buttonDisabled: boolean;
+  @Input() buttonIcon: string;
+
+  @Output() handleClick = new EventEmitter<boolean>();
```

Comment:
It looks like this `Output` should emit a type of `boolean`, however when `emit` is called down below on line `14` we're not passing anything in the params so it'll always be false. I don't think this _really_ matters, but it might be worthwhile to change this to emit a type of `void`. Thoughts?

Reply:
Agreed!